snmp/datadog_checks/snmp/data/default_profiles/barracuda-cloudgen.yaml

```
@@ -0,0 +1,206 @@
+extends:
+  - barracuda.yaml
+
+sysobjectid:
+  - 1.3.6.1.4.1.10704.1.*
+
+metrics:
+  - MIB: PHION-MIB
+    table:
+      OID: 1.3.6.1.4.1.10704.1.0
+      name: boxServices
+    symbols:
+      - name: phion.boxServices
+        constant_value_one: true
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.10704.1.0.1.1
+          name: boxServiceName
+        tag: box_service_name
```

In newer profiles, when automatically adding a prefix like `phion.`, tags are also prefixed. 

I think for consistency, we can maybe add prefix to tags too in this profile. (Opinion might vary about wether we need to prefix tags or not, but I think it's safer to have a prefix to avoid possible collisions, it's likely not the case for this profile though)

Example:
```suggestion
        tag: phion_box_service_name
```